### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by luftedar**
**Hello Microverse Project**

In this project:

- Added HTML & CSS with header, its style.
- Added test.md 
- Updated README.md
- Added Linters

### CODE REVIEW

File: .gitignore/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (carlos-ssh) [inline .gitignore/workflows/linters.yml]: **Kindly enable your linters in your pull request.**

- Linter should be set up correctly for this repository.
- Linter result should be green. AKA: Linter should report NO errors.
- There should be no custom changes to linter config (as an attempt to cheat linter error reports).

File: README.md [hunk 2]
```diff
@@ -1,59 +1,25 @@
...
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- HTML & CSS
+- GitHub Workflows
```
Comment (carlos-ssh) [inline README.md:14]: Kindly fill information removed like how to download your repo, install your project locally how to use it which requirements are needed to keep on mind before to install. Probably testing and deployment isn't required.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @luftedar , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (luftedar) [reply README.md:12]: Hello!

**Comment by luftedar** [comment]
In this update:

- Fixed linter request
- updated README.md file

File: .gitignore/test.md [hunk 3]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (dasileker) [inline .gitignore/test.md]: - [ ] The **.gitignore** is not a **folder** but a **_file_** see the screenshots 👇.

Capture web_14-9-2021_16311_github com
mmmm
lmlml

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @luftedar,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luftedar** [comment]
**Second Update**

In this update:

    - Updated .gitignore
    - Updated test.md

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔ 💪 </h1>
Hi @luftedar, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**